client/app/models/land-use.js

```diff
@@ -0,0 +1,81 @@
+import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
+
+export default class LandUseFormModel extends Model {
+  @belongsTo('package', { async: false })
+  package;
+
+  @hasMany('applicant', { async: false })
+  applicants;
+
+  @hasMany('land-use-action', { async: false })
```

Comment:
I think the land use action model name should be `landuse-action`, and the relationship `landuseActions`?